prometheus/datadog_checks/prometheus/data/conf.yaml.example

```diff
@@ -1,89 +1,119 @@
 init_config:
-  # Leave empty
 
 instances:
-  # The prometheus endpoint to query
-  #
-  # - prometheus_url: http://service/prometheus
 
-  #   The namespace to be appended before all metrics namespace
+  ## @param prometheus_url - string - required
+  ## The prometheus endpoint to query.
   #
-  #   namespace: "service"
+  - prometheus_endpoint: http://localhost:10055/metrics
 
-  #  List of metrics to be fetched from the prometheus endpoint, if there's a value it'll be renamed
-  #  This list should contain at least one metric
+  ## @param namespace - string - optional
+  ## The namespace to be appended before all metrics namespace
   #
-  #   metrics:
-  #     - processor: cpu
-  #     - memory: mem
-  #     - io
+  #  namespace: <SERVICE_NAME>
 
-  # Prefix for exposed Prometheus metrics
+  ## @param metrics - list of key:value elements - optional
```

Comment:
I guess as the list requires at least one metric it's required?

It's a bit tricky as there's no valid default